elements/pl-multiple-choice/pl-multiple-choice.py

```diff
@@ -1,33 +1,69 @@
 import json
-import math
 import pathlib
 import random
+from collections import Counter
+from enum import Enum
+from itertools import chain, count
+from typing import List, NamedTuple, Optional, Tuple
 
 import chevron
-import lxml.html
+import lxml
 import prairielearn as pl
+from typing_extensions import assert_never
+
+
+class DisplayType(Enum):
+    INLINE = "inline"
+    BLOCK = "block"
+    DROPDOWN = "dropdown"
+
+
+class AotaNotaType(Enum):
+    FALSE = 1
+    RANDOM = 2
+    INCORRECT = 3
+    CORRECT = 4
+
+
+class SortType(Enum):
```

Comment:
There has been some discussion about which types of sorting should be supported. We should decide on that before this gets merged.